.travis.yml

```diff
@@ -4,13 +4,5 @@ android:
   components:
     - tools
     - platform-tools
-
-    # The SDK version used to compile your project
     - android-26
-
-    # The BuildTools version used by your project
     - build-tools-26.0.2
-
-# as per http://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/
```

Comment:
Based on my tests this setting does speed up travis build by about 20%

Reply:
Hmm very interesting. When did you do these tests? Travis has deprecated this feature and moved all builds to container-based infrastructure a while ago, see here: https://docs.travis-ci.com/user/reference/overview/
What this means is that `sudo` *defaults* to `false` - on certain environments you can *manually* set it to `required`, but here I don't see how it can make a difference..

Reply:
Sorry for not explaining in the PR description!